sourcegraph/sourcegraph
# (feat) add repo match search results aggregation by repo metadata

Part of https://github.com/sourcegraph/pr-faqs/issues/96.

This PR introduces new aggregation mode over repository metadata. It works for `select:repo` and shows number of repositories for a given key-value pair metadata.

## Test plan
- `sg start`
- Set `repository-metadata` feature flag to `true`
- Use search with additional `select:repo` and check that aggregation panel had new "Repo metadata" option

## Screenshots

https://github.com/sourcegraph/sourcegraph/assets/6717049/8d3b4064-71a3-49aa-aaf0-9196e929fbe9
